### PR TITLE
MAT-8582 Polling to fetch latest case every 5 seconds when validation…

### PR DESCRIPTION
**Pull request by RohitKandimalla**
… status is pending

## MADiE PR

Jira Ticket: [MAT-8582](https://jira.cms.gov/browse/MAT-8582)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/testCases/components/editTestCase/qiCore/EditTestCase.tsx [hunk 1]
```diff
@@ -530,6 +553,12 @@ const EditTestCase = (props: EditTestCaseProps) => {
         testCase,
         measureId
       );
+      // initiate polling for validation response
+      if (updatedTestCase.testCaseValidationStatus === "Pending") {
```
Comment (sb-cecilialiu) [inline src/components/editMeasure/testCases/components/editTestCase/qiCore/EditTestCase.tsx:557]: what if the status is "Validating"?

Comment (RohitKandimalla) [reply src/components/editMeasure/testCases/components/editTestCase/qiCore/EditTestCase.tsx:557]: As soon as a user updatesTestCase the validationStatus will always be Pending, it will change to Validating in consecutive calls which will be captured in loadTestCase method

**Review by sb-cecilialiu — APPROVED** [review]
